modules/Microsoft.EventGrid/subscriptions/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,134 @@
+param eventTopicName string
+
+@description('Required. The name of the Service Bus Namespace to create.')
+param serviceBusName string
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+    name: virtualNetworkName
+    location: location
+    properties: {
+        addressSpace: {
+            addressPrefixes: [
+                '10.0.0.0/24'
+            ]
+        }
+        subnets: [
+            {
+                name: 'defaultSubnet'
+                properties: {
+                    addressPrefix: '10.0.0.0/24'
+                }
+            }
+        ]
+    }
+}
+
+resource privateDNSZone 'Microsoft.Network/privateDnsZones@2020-06-01' = {
+    name: 'privatelink.eventgrid.azure.net'
+    location: 'global'
+
+    resource virtualNetworkLinks 'virtualNetworkLinks@2020-06-01' = {
+        name: '${virtualNetwork.name}-vnetlink'
+        location: 'global'
+        properties: {
+            virtualNetwork: {
+                id: virtualNetwork.id
+            }
+            registrationEnabled: false
+        }
+    }
+}
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+    name: managedIdentityName
+    location: location
+}
+
+resource eventTopic 'Microsoft.EventGrid/topics@2022-06-15' = {
+    name: eventTopicName
+    location: location
+    properties:{
+        disableLocalAuth: false
+        publicNetworkAccess: 'Enabled'
+    }
+    identity: {
+        type: 'UserAssigned'
+        userAssignedIdentities: {
+            '${managedIdentity.id}': {}
+        }
+    }
+}
+
+resource serviceBus 'Microsoft.ServiceBus/namespaces@2021-11-01' = {
+    name: serviceBusName
+    location: location
+    properties: {
+        sku: {
+            name: 'Standard'
+            tier: 'Standard'
+            capacity: 1
+        }
+        zoneRedundant: false
+    }
+}
+
+resource serviceBus_Queue 'Microsoft.ServiceBus/namespaces/queues@2021-11-01' = {
+    name: '${serviceBus.name}/queue1'
+    properties:{
+        lockDuration: 'PT1M'
+        maxSizeInMegabytes: 1024
+        requiresDuplicateDetection: false
+        requiresSession: false
+        defaultMessageTimeToLive: 'P10675199DT2H48M5.4775807S'
+        deadLetteringOnMessageExpiration: false
+        duplicateDetectionHistoryTimeWindow: 'PT10M'
+        maxDeliveryCount: 10
+        enableBatchedOperations: true
+        status: 'Active'
+        autoDeleteOnIdle: 'P10675199DT2H48M5.4775807S'
+    }
+}
+
+resource serviceBus_Topic 'Microsoft.ServiceBus/namespaces/topics@2021-11-01' = {
+    name: '${serviceBus.name}/topic1'
+    properties:{
+        maxSizeInMegabytes: 1024
+        requiresDuplicateDetection: false
+        defaultMessageTimeToLive: 'P10675199DT2H48M5.4775807S'
+        duplicateDetectionHistoryTimeWindow: 'PT10M'
+        enableBatchedOperations: true
+        status: 'Active'
+        autoDeleteOnIdle: 'P10675199DT2H48M5.4775807S'
+    }
+}
+
```

Comment:
It appears we're returning a lot more outputs than we actually used. I'd suggest removing the 'noise'

Reply:
Done in new commit